server.js

```diff
@@ -0,0 +1,73 @@
+var express = require('express');
```

Comment:
Did you mean to use ES6 import syntax here? Looks like you used it below, so might be good to change this one for consistency.